modules/ROOT/pages/vm/vm-reference.adoc

```diff
@@ -170,12 +172,14 @@ The queue in which the content is published cannot contain a Listener (`<vm:list
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| Content a| Any |  +++the content to be published+++ |  +++#[payload]+++ | {nbsp}
+| Content a| Any |  +++The content to be published+++ |  +++#[payload]+++ | {nbsp}
```

Comment:
Did this get run through the cleanup script? That usually deletes all occurrences of {nbsp}